content/02.main-text.md

```diff
@@ -203,7 +203,7 @@ The architecture of the plugins and themes is designed to provide authors with e
 
 ![
 **Examples of the various Manubot plugins, illustrating their functionality and usefulness.**
-Screenshots were taken from existing manuscripts made with Manubot: [Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/v/fd7acb7ed0108c920da56f84819ce13f02f68aa8/) and [TPOT-DS](https://trang1618.github.io/tpot-ds-ms/), available under the [CC BY 4.0 License](https://creativecommons.org/licenses/by/4.0/ "Creative Commons Attribution 4.0 International License").
+Screenshots were taken from existing manuscripts made with Manubot: [Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/v/fd7acb7ed0108c920da56f84819ce13f02f68aa8/) and [TPOT-FSS](https://trang1618.github.io/tpot-fss-ms/), available under the [CC BY 4.0 License](https://creativecommons.org/licenses/by/4.0/ "Creative Commons Attribution 4.0 International License").
```

Comment:
Should we do something for the TPOT-DS link in the Figure 3 caption in this pull request?  That link is currently broken.

Reply:
Yes this line that you commented on changes it to https://trang1618.github.io/tpot-fss-ms/, which currently works.

Reply:
Sorry, not sure what I was thinking.  I thought this was the reference in the example manuscript section, which you also changed.